src/python/pants/rules/core/filedeps.py

```diff
@@ -16,24 +17,36 @@ class Filedeps(LineOriented, Goal):
   closure of targets are also included.
   """
 
-  # TODO: Until this implements more of the options of `filedeps`, it can't claim the name!
+  # TODO: Until this implements `--globs`, this can't claim the name `filedeps`!
```

Comment:
I think that to implement `--globs`, this could inspect `sources.filespec`, potentially?

Reply:
That worked! Thanks!